extensions/iam/decentralized-identity/registration-service/src/main/java/org/eclipse/dataspaceconnector/iam/registrationservice/crawler/CrawlerJob.java

```diff
@@ -109,7 +115,8 @@ private List<String> getDidSuffixesForType(String type, TypeManager typeManager)
         try (var response = client.newCall(request).execute()) {
             if (response.isSuccessful()) {
                 var json = Objects.requireNonNull(response.body()).string();
-                var typeReference = new TypeReference<List<String>>() {};
+                var typeReference = new TypeReference<List<String>>() {
+                };
```

Comment:
nit: maybe not needed.
```suggestion
                var typeReference = new TypeReference<List<String>>() {};
```